tools/npm/garaga_ts/src/node/api.ts

```diff
@@ -20,10 +31,16 @@ export type MsmCalldataBuilderOptons = Partial<{
 }>;
 
 export function msmCalldataBuilder(points: G1Point[], scalars: bigint[], curveId: CurveId, options: MsmCalldataBuilderOptons = {}): bigint[] {
-  const values = points.reduce<bigint[]>((values, point) => values.concat(point), []);
-  const includeDigitsDecomposition = options.includeDigitsDecomposition ?? true;
-  const includePointsAndScalars = options.includePointsAndScalars ?? true;
-  const serializeAsPureFelt252Array = options.serializeAsPureFelt252Array ?? false;
-  const risc0Mode = options.risc0Mode ?? false;
+  const values = flatten(points);
+  const includeDigitsDecomposition = options.includeDigitsDecomposition || true;
+  const includePointsAndScalars = options.includePointsAndScalars || true;
+  const serializeAsPureFelt252Array = options.serializeAsPureFelt252Array || false;
+  const risc0Mode = options.risc0Mode || false;
```

Comment:
Be careful when merging to keep the ?? . See https://github.com/keep-starknet-strange/garaga/pull/205

Reply:
Wow, it was a bug and I did not see I was overriding it. Thanks!